workloads/kube-burner/workloads/networkpolicy/clusterrolebinding.yml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pods-list-crb-1
+subjects:
+- kind: Group
+  name: system:authenticated
```

Comment:
.. granted to all users.

Reply:
If this role binding is for pod-scraper, it looks like only view perms are required on pods and networkpolicies inside the namespace.

Not a requirement, just a nit